readme.md

```diff
@@ -175,8 +176,9 @@ The [DICOM Standard](https://www.dicomstandard.org/) is _the_ international stan
 #### Visualization
 
 - [dicom-ecg-plot](https://github.com/marcodebe/dicom-ecg-plot) - Plot ECG data from DICOM ([demo](https://ecg.galliera.it/)).
+- [FSLeyes](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes - A Python-based cross-platform visualization tool for NIfTI and DICOM images.
 - [OnkoDICOM](https://github.com/didymo/OnkoDICOM) - DICOM-RT viewer with enhanced capabilities that make it useful for research in the field of Radiation Oncology. 
-- [MRIcroGL](https://github.com/rordenlab/MRIcroGL) - A cross-platform tool for viewing DICOM and [NIfTI](https://nifti.nimh.nih.gov/) format images. It provides a drag-and-drop user interface as well as a scripting language.
+- [MRIcroGL](https://github.com/rordenlab/MRIcroGL) - A cross-platform tool that supports many voxel-based image formats including DICOM. This natively compiled application provides a drag-and-drop user interface as well as Python scripting integration.
```

Comment:
```suggestion
- [MRIcroGL](https://github.com/rordenlab/MRIcroGL) - A cross-platform tool that supports many voxel-based image formats including DICOM. This natively compiled application provides a drag and drop user interface as well as Python scripting integration.
```
Changed the previously existing “drag-and-drop” bit to conform with its previous styling as “drag and drop”, which also matches most the common styling I found in a Google search for the term.